packages/cubejs-server-core/core/index.js

```diff
@@ -41,6 +41,29 @@ const checkEnvForPlaceholders = () => {
   }
 };
 
+const devLogger = (type, message, error) => {
+  const colors = {
+    red: '31', // ERROR
+    green: '32', // INFO
+    blue: '34', // TRACE
+  }
+  const withColor = (str, color = colors.green) => `\u001b[${color}m${str}\u001b[0m`;
+  const format = json => JSON.stringify(json, null, 2);
+
+  if ([ // INFO list
+    'Performing query',
+    'Performing query completed',
+  ].includes(type)) {
+    if (error) console.log(`${withColor(type, colors.blue)}: ${error}`);
+    else console.log(`${withColor(type, colors.blue)}: ${format(message)}`);
+    return;
+  }
```

Comment:
Also let's output all other messages for `INFO` but just without log event props. This way we can keep console clean and still readable.

Reply:
OK, I get it.